docs/manual/manual.adoc

```diff
@@ -451,6 +459,15 @@ is a CVE you can click it to display its metadata from the official NVD
 database (requires internet connection). OpenSCAP doesn't provide
 metadata for other types of identifiers.
 
+Another place, where these identifiers are located is machine-readable Result
+Datastream files.
+
```

Comment:
It would be better to tell the reader what the command does just before you write the command. I suggest something like "This file can be generated during the scan by adding --results-arf option."